lib/libyui_client/widgets/base.rb

```diff
@@ -18,9 +18,11 @@ def initialize(widget_controller, filter)
       # @example Check if button with id 'test' exists
       #   app.button(id: 'test').exists? # true
       def exists?
-        LibyuiClient.logger.info("Does #{class_name} exist: #{@filter}")
-        true if @widget_controller.find(@filter, timeout: 0, interval: @interval)
+        @widget_controller.find(@filter, timeout: 0, interval: @interval).body
```

Comment:
Could you please add also logger.info before the `find`. To first log what the client is trying to do, then log the result. Something like `Checking if #{class_name} with #{@filter} exists`.

Reply:
fixed